src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -33,6 +33,7 @@
 import io.fabric8.elasticsearch.plugin.OpenshiftRequestContextFactory.OpenshiftRequestContext;
 import io.fabric8.elasticsearch.plugin.PluginSettings;
 import io.fabric8.elasticsearch.plugin.kibana.KibanaSeed;
+import io.fabric8.elasticsearch.plugin.rest.RestChannelInterceptor;
```

Comment:
Is this api available in ES 2.x e.g. can we backport this to earlier releases?

Reply:
This was code I inserted and may be something we could backport but do we have need? 3.11 is considered our long term release which is based on 5.x.  We should refrain from backporting unless we have a good reason (e.g. bug, etc)